tests/utils.go

```diff
@@ -996,24 +996,15 @@ func NewRandomVMI() *v1.VirtualMachineInstance {
 }
 
 func NewRandomVMIWithNS(namespace string) *v1.VirtualMachineInstance {
-	vmi := v1.NewVMIReferenceFromNameWithNS(namespace, libvmi.RandName())
-	vmi.Spec = v1.VirtualMachineInstanceSpec{Domain: v1.DomainSpec{}}
-	vmi.Spec.Domain.Resources.Requests = k8sv1.ResourceList{}
-	vmi.TypeMeta = metav1.TypeMeta{
-		APIVersion: v1.GroupVersion.String(),
-		Kind:       "VirtualMachineInstance",
-	}
-
-	t := defaultTestGracePeriod
-	vmi.Spec.TerminationGracePeriodSeconds = &t
-
 	// To avoid mac address issue in the tests change the pod interface binding to masquerade
 	// https://github.com/kubevirt/kubevirt/issues/1494
-	vmi.Spec.Domain.Devices = v1.Devices{Interfaces: []v1.Interface{{Name: "default",
-		InterfaceBindingMethod: v1.InterfaceBindingMethod{
-			Masquerade: &v1.InterfaceMasquerade{}}}}}
+	vmi := libvmi.New(
+		libvmi.RandName(),
+		libvmi.WithMasqueradeNetworking()...,
```

Comment:
`WithMasqueradeNetworking` is confusing. It includes also cloud-init network config, which I think you should not have here.

Reply:
I don't understand, but let me try it. Another option is to drop Masquerade altogether, given the fact that https://github.com/kubevirt/kubevirt/issues/1494 is resolved in libvirt. But this should way for another PR and further discussion.

Reply:
The one you added includes cloud-init configuration of the guest, while the one that existed until now does not.